include/alpaka/rand/TinyMT/tinymt32.h

```diff
@@ -352,9 +352,9 @@ void tinymt32_init(tinymt32_t * random, uint32_t seed) {
  */
 void tinymt32_init_by_array(tinymt32_t * random, uint32_t init_key[],
                             int key_length) {
-    const unsigned int lag = 1;
-    const unsigned int mid = 1;
-    const unsigned int size = 4;
+    constexpr unsigned int lag = 1;
+    constexpr unsigned int mid = 1;
+    constexpr unsigned int size = 4;
```

Comment:
This is third-party code; I'd like to avoid to change it.

Reply:
I agree. Is there a particular reason you need to have these files changes? If static analysis flags those, we may be able to disable it for this file.